mslice/widgets/dataloader/dataloader.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import (absolute_import, division, print_function)
+
+import os
+
+from functools import partial
+
+from qtpy.QtWidgets import QWidget, QFileSystemModel, QAbstractItemView, QMessageBox
+from qtpy.QtCore import Signal, QDir
+
+from mslice.presenters.data_loader_presenter import DataLoaderPresenter
+from mslice.util.qt import load_ui
+from .inputdialog import EfInputDialog
+
+
+class DataLoaderWidget(QWidget): # and some view interface
+
+    error_occurred = Signal('QString')
+
+    def __init__(self, parent=None):
+        QWidget.__init__(self, parent)
+        load_ui(__file__, 'dataloader.ui', self)
+
+        self.file_system = QFileSystemModel()
+        self.directory = QDir("C:/")
```

Comment:
This doesn't really make sense for Linux & OSX. How about we start in `os.path.expanduser('~')` (basically $HOME)?

I also noticed a disparity on my system that when I started `mslice` it displayed `/` in the edit box but then when I hit "refresh" it went to my current working directory. I assume it should refresh what is currently displayed?